app/Jobs/GetBlockDetailsJob.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Appeal;
+use App\MwApi\MwApiGetter;
+use App\MwApi\MwApiExtras;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+
+class GetBlockDetailsJob implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
+
+    private $appeal;
+
+    public function __construct(Appeal $appeal)
+    {
+        $this->appeal = $appeal;
+    }
+
+    public function handle()
+    {
+        if ($this->appeal->wiki === 'global') {
+            $details = MwApiExtras::getGlobalBlockInfo($this->appeal->appealfor);
+        } else {
+            $details = MwApiExtras::getBlockInfo($this->appeal->wiki, $this->appeal->appealfor);
```

Comment:
There does not seem to be any handling for appeal[14] (in the python script - db equivalent) or hiddenip (the db column).

Am I missing it?

Reply:
Fixed, thanks.

Reply:
Unresolving so I can look at the change more easily when I review it